conanfile.py

```diff
@@ -7,7 +7,8 @@
 
 class FmtConan(ConanFile):
     name = "fmt"
-    version = "4.1.0"
+    lib_version = "4.1.0"
+    version = "4.1.0-r1"
```

Comment:
Why did you change the version to r1?

Reply:
I don't know your policy when you release an updated recipe, but not the actual released version. We append a "release" version here. I guess you lean on the "--outdate" of conan  install ?

Reply:
so you are asking for package version, now I see.

We merge all PR over **testing** branch and after to be approved, we merge over **stable**. Only the stable branch is uploaded to Bintray.

 About package version, unfortunately Conan does not support yet and only `self.version` is collected to create the package version. So we don't patch the package version. And how does user know if package is outdated? Well there is `conan install --update` and another command to list outdated packages, but there is no patch number version to checkout if you want to downgrade.